category/fold.lean

```diff
@@ -119,8 +119,8 @@ traverse (foldl.mk ∘ flip f) xs x
 def foldr (f : α → β → β) (x : β) (xs : t α) : β :=
 traverse (foldr.mk ∘ f) xs x
 
-def to_list (xs : t α) : list α :=
-traverse (foldr.mk ∘ list.cons) xs []
+def to_list : t α → list α :=
+foldr list.cons []
```

Comment:
I'm joking. Because your initial commit was worded as a massacre and now you're coming back for more??

But seriously, I'm truly grateful for these improvements.

Reply:
> I'm joking. Because your initial commit was worded as a massacre and now you're coming back for more??

Ah. 😆 🔨🔪 

> But seriously, I'm truly grateful for these improvements.

I'm glad!